.github/workflows/CI-Fast-Tooling

```diff
@@ -0,0 +1,28 @@
+name: CI-Fast-Tooling
+
+on:
+  push:
+    branches: [ release ]
+  pull_request:
+    branches: [ release ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
Is there a point to using a strategy matrix if there is only one option we are running the build against?